File: xdr/backends/python/templates/typedef.py
class ${typedef.declaration.name}(object):
    @classmethod
    def pack_into(self, packer, obj):
:: include_indented("_pack.py", indent=8, m=typedef.declaration, src="obj")

    @classmethod
    def unpack_from(cls, unpacker):
:: include_indented("_unpack.py", indent=8, m=typedef.declaration, dst="obj")
        return obj

File: xdr/backends/lua/templates/typedef.lua
function public.read_${typedef.declaration.name}(reader)
    local obj
:: include("_unpack.lua", m=typedef.declaration, dst="obj")
    return obj
end

function public.write_${typedef.declaration.name}(writer, obj)
:: include("_pack.lua", m=typedef.declaration, src="obj")
end
